<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>miniwindows</class>
 <widget class="QWidget" name="miniwindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    background-color: #FFFEF9;
}

QMenuBar {
    background-color: #FFFCF3;
}

QMenu {
    background-color: #FFFCF3;
}

QMenu::item {
    padding: 5px 20px;
    background-color: transparent;
}

QMenu::item:selected {
    background-color: #f8b156;
}

QMenu::separator {
    height: 1px;
    background-color: #f7931e;
    margin: 5px 0px 5px 0px;
}

QMenuBar::item {
    padding: 5px 20px;
    background-color: transparent;
}

QMenuBar::item:selected {
    background-color: #f8b156;
}

QMenuBar::item:pressed {
    background-color: #DAA520;
}


QLineEdit {
    border: 2px solid #f7931e;
    border-radius: 3px;
    background-color: #FFFCF3;
    padding: 2px;
    font-size: 16px;
}


QTextEdit {
    border: 2px solid #f7931e;
    border-radius: 3px;;
    padding: 5px;
    font-size: 10px;
    background-color: #FFFCF3;
}

QTableView {
    background-color: #FFFCF3;
}

QPushButton {
    background-color: #FFBC33;
    color: #ffffff;
    border: none;
    border-radius: 2px;
    padding: 2px 2px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #f8b156;
}

QPushButton:pressed {
    background-color: 	#DAA520;
}


QComboBox {
    border: none;
    border-radius: 20px;
    background-color: #FFBC33;
    color: #ffffff;
    font-size: 16px;
    font-weight: bold;
    padding: 8px 20px;
    padding-right: 4px; /* 右侧留出一定空间 */
    selection-background-color: #f8b156; /* 选中项背景色 */
    selection-color: #ffffff; /* 选中项前景色 */
    outline: none;
}

QComboBox:hover {
    background-color: #f8b156;
}

QComboBox::drop-down {
    width: 0; /* 去掉下拉箭头 */
}

QCheckBox {
    color: #FFBC33;
    font-size: 16px;
    font-weight: bold;
    padding-left: 20px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
}

QCheckBox::indicator::unchecked {
    border: 2px solid #FFBC33;
    background-color: #FFFEF9;
}

QCheckBox::indicator::unchecked:hover {
    border: 2px solid #f8b156;
}

QCheckBox::indicator::checked {
    border: 2px solid #FFBC33;
    background-color: #FFBC33;
}

QCheckBox::indicator::checked:hover {
    background-color: #f8b156;
}
</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget::item {
	padding: 0px;
	border: 0;
}</string>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_location">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选中方块：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_block">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>方块：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_toward">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>朝向：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_redstone">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>红石：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_condition">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>条件：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_execute">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>执行：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_delay">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>延迟：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_delay_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_delay_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>命令</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_notes">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_cmd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始导入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>位置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
